docs/change_log/2017-07-24-new-feature-audit-logs.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "New Feature: Audit Logs"
+date: "2017-07-24T07:00:00.000Z"
+breaking: false
```

Comment:
uhhh should we leave breaking out unless it's `true`? ideally this is happens infrequently